### DISCUSSION TITLE
Source code of release 1.4.3

### DISCUSSION THREAD

**Discussion by Marv51** [microsoft/microsoft-ui-xaml#9108] [General]
Hi, 

The `winui3/release/1.4.3` and the `winui3/release/1.4.2` tags currently point towards the same commit (7198c4e).

I would assume that the source for these two releases is not identical.

So am I misunderstanding what the current process for source availability is? Or was the source for the 1.4.3 release not updated correctly?

Is this a mistake or can someone explain it?

Thanks!

--
/cc: @duncanmacmichael

**Comment by evelynwu-msft** [comment]
@Marv51 Thanks for bringing this to our attention! The tags have been fixed up and now point to the correct commits.
